hoot-core-test/src/test/cpp/hoot/core/io/OsmApiWriterTestServer.cpp

```diff
@@ -31,13 +31,6 @@
 #include <hoot/core/io/OsmApiWriter.h>
 #include <hoot/core/util/Log.h>
 
-//  Standard
-#include <memory>
-
-//  Boost
-#include <boost/bind.hpp>
-#include <boost/asio.hpp>
-
```

Comment:
Almost all of the deleted code was actually moved to the `HttpTestServer` class below.